bin/redmapper_batch.py

```diff
@@ -278,10 +280,45 @@ def load_batchconfig(filename):
         jf.write(parsl_script)
 
     elif (batchconfig[batchmode]['batch'] == 'slurm'):
-        raise NotImplementedError("Basic slurm submission not implemented yet.  Use parsl")
+
+        mem_per_node = int(batchconfig[batchmode]['mpn'])
+        pixels_per_node = int(np.floor(mem_per_node / memory))
+        total_pixels = hpix_run.size - 1
```

Comment:
Why `- 1`?

Reply:
I think this was a typo and had no affect on my runs.